src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -581,7 +581,7 @@ private static String getMessageBundle(String className) {
     protected static String[] removeSuppressed(String[] actualViolations,
                                                String... suppressedViolations) {
         final List<String> actualViolationsList =
-            Arrays.stream(actualViolations).collect(Collectors.toList());
+            Arrays.stream(actualViolations).collect(Collectors.toCollection(ArrayList::new));
         actualViolationsList.removeAll(Arrays.asList(suppressedViolations));
```

Comment:
unrelated but actualViolationsList should  be Set.

Reply:
I am not sure it can be since it is a String. I think order matters, as it is the order of violations. Other methods are list too. I think this should be a separate PR to confirm.